metricbeat/mb/mb.go

```diff
@@ -84,6 +90,42 @@ type EventsFetcher interface {
 	Fetch() ([]common.MapStr, error)
 }
 
+// Reporter is used by a MetricSet to report events, errors, or errors with
+// metadata. The methods return false iff publishing failed because the
```

Comment:
[iff](https://en.wikipedia.org/wiki/If_and_only_if) is shorthand for "if and only if". I thought it was more common, maybe only common in math. I'll expand it to "if and only if".